bodhi-server/bodhi/server/metadata.py

```diff
@@ -95,6 +95,73 @@ def modifyrepo(comp_type, compose_path, filetype, extension, source, zchunk):
             repodata = os.path.join(repo_path, arch, 'os', 'repodata')
         insert_in_repo(comp_type, repodata, filetype, extension, source, zchunk)
 
+class FrequencyUpdateInfoMetadata(object):
+    """
+    This class represents the frequencyupdateinfo.json metadata
+    """
+
+    def __init__(Release, request, db, compose_dir, Update, UpdateStatus, ReleaseState):
+        """
+        Initialize the FrequencyUpdateInfoMetadata object.
+
+        Args:
+            release (bodhi.server.models.Release): The Release that is being composed.
+            request (bodhi.server.models.UpdateRequest): The Request that is being composed.
+            db (): A database session to be used for queries.
+            composedir (str): A path to the composedir.
+            update (bodhi.server.models.Update):
+            update_status (bodhi.server.models.UpdateStatus):
+            release_state (bodhi.server.models.ReleaseState):
+        """
+        updates = update.query.filter(
+            update.status.in_(
+                [update_status.stable])
+        ).filter(
+            update.release_id == release.id
+        ).filter(
+            release.state.in_([
+                release_state.current,
+                release_state.pending,
+            ])
+        ).order_by(
+            # Check the older updates first so there is more time for the newer to
+            # get their test results
+            update.id.asc()
+        )
+        modified_updates = []
+        for update in updates:
+            #Getting the specific fields from data required for processing
+            simplified_update = {
+                "release" : {
+                    "name" : update.release.name 
+                    "version" : update.release.version
+                }
+                "alias" : update.alias
+                "date_stable" : update.date_stable
+                "builds" : [
+                ]
+            } 
+            for build in update.builds:
+                simplified_build = {"nvr_name" : build.nvr_name()}
+                simplified_update["builds"].push(simplified_build)
```

Comment:
`"builds" : [{"nvr_name" : build.nvr_name()} for build in update.builds]`

And missing commas between dict elements.